.github/workflows/buildCSharp.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: "Checkout develop Branch"
         uses: actions/checkout@v3
         with:
-          ref: develop
+          ref: master
```

Comment:
This is temporary, trying to generate the nuget for 3.6.0 again at https://github.com/NREL/OpenStudio/actions/runs/4989940843

Also, I don't think we generally always want to build develop, do we @tijcolem ? We should omit the `ref` and let Github figure out what to build: a push to master would build master, a push of a tag would build the tag, a PR to master would build the resulting commit of the PR merged into master. Doesn't that make more sense?

Reply:
https://github.com/NREL/OpenStudio/actions/runs/4989940843 worked. @tijcolem

Reply:
@MingboPeng I think you manually reupload these at the platform specific nuget endpoint?
eg https://www.nuget.org/packages/NREL.OpenStudio.macOS-x64, https://www.nuget.org/packages/NREL.OpenStudio.linux-x64, https://www.nuget.org/packages/NREL.OpenStudio.win

Reply:
Hi @jmarrec yes, I manually uploaded those packages.

Reply:
@MingboPeng If you want to do that again, https://github.com/NREL/OpenStudio/actions/runs/4989940843 if where you could get them.